src/analytics/lambdas/scan-metadata-workflow/store-metadata-into-ddb.ts

```diff
@@ -421,6 +422,7 @@ async function updateEventParameterItem(itemsMap: Map<string, any>, key: string,
     addSetIntoAnotherSet(eventNameSet, convertToSet(record[7].stringValue));
     item.summary.associatedEvents = Array.from(eventNameSet);
   }
+  item.version = METADATA_V3_VERSION;
```

Comment:
We should add this version to KEY of table, like `prefix=EVENT#${projectId}#${appId}#${version}`